types/got/v8/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // TODO
```

Comment:
What is a mean?

Reply:
I just copied it from the original (v8) type definition. I guess I can just remove the todo.